commit 2c2e54000ffc34071da3af838275bb8bf12f7fb0
Author: Phil Elwell <phil@raspberrypi.org>
Date:   Tue Feb 24 13:40:50 2015 +0000

    pinctrl-bcm2835: Fix interrupt handling for GPIOs 28-31 and 46-53
    
    Contrary to the documentation, the BCM2835 GPIO controller actually has
    four interrupt lines - one each for the three IRQ groups and one common. Rather
    confusingly, the GPIO interrupt groups don't correspond directly with the GPIO
    control banks. Instead, GPIOs 0-27 generate IRQ GPIO0, 28-45 GPIO1 and
    46-53 GPIO2.
    
    Awkwardly, the GPIOS for IRQ GPIO1 straddle two 32-entry GPIO banks, so it is
    cleaner to split out a function to process the interrupts for a single GPIO
    bank.
    
    This bug has only just been observed because GPIOs above 27 can only be
    accessed on an old Raspberry Pi with the optional P5 header fitted, where
    the pins are often used for I2S instead.

diff --git a/drivers/pinctrl/bcm/pinctrl-bcm2835.c b/drivers/pinctrl/bcm/pinctrl-bcm2835.c
index 6351fe7f8e31..28745af5aadf 100644
--- a/drivers/pinctrl/bcm/pinctrl-bcm2835.c
+++ b/drivers/pinctrl/bcm/pinctrl-bcm2835.c
@@ -1109,6 +1109,7 @@ static struct platform_driver bcm2835_pinctrl_driver = {
 	.remove = bcm2835_pinctrl_remove,
 	.driver = {
 		.name = MODULE_NAME,
+		.owner = THIS_MODULE,
 		.of_match_table = bcm2835_pinctrl_match,
 	},
 };
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1167_2c2e54000ffc34071da3af838275bb8bf12f7fb0.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1167_2c2e54000ffc34071da3af838275bb8bf12f7fb0.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
